README.md

```diff
@@ -1,2 +1,34 @@
 # hw01
-Repository for first CFSS homework assignment
+# Nora C. Nickels  
+
+### ![](https://0.academia-photos.com/30174041/13103076/14424279/s200_nora.nickels.png) 
+Hi, my name is Nora and I am a Ph.D. student taking this class to learn more about programming and statistics. For my master's project two years ago, I studied how stress affects decision making, by bringing subjects into the lab, stressing them out (very rude, I know), measuring their hormone levels, and looking at their performance on decision making tasks. I was taught by fellow graduate students to use EPrime in order to create computer decision making tasks for my participants to take. I really enjoyed learning to code and program in EPrime, but I was using EPrime in the "click and drag" meantality similar to that of SPSS. I hope to use this class to learn more about the programming behind the tasks I'm creating and the statistical analyses that I'm using. 
+
+### Affiliations
+
+* Department of Comparative Human Development
+* University of Chicago
+* Behavioral Biology Lab
+* Institute for Mind and Biology 
+
+### Bio
+
+*I am a third year Ph.D. student in the Department of Comparative Human Development at the University of Chicago, working in the Behavioral Biology Lab under Dr. Dario Maestripieri. My general research interests include the neuroendocrine bases of human decision-making and cooperation, behavioral and biological responsiveness to social stimuli and psychosocial stress, and individual differences in behavior in relation to sex and stress hormones.*
+
+
+**Websites:**
+
+https://sites.google.com/site/noracnickels/
+
+https://scholar.google.com/citations?user=7XtEhe0AAAAJ&hl=en
+
+https://imb.uchicago.edu/directory/nora-nickels
+
+http://chicago.academia.edu/NoraNickels
+
+## 
+
+> "But with my ego, I could stand there in a speedo and be looked at like a f***ing hero"
+>
+-Kanye West
+
```

Comment:
**Coding style**  +
 **Coding strategy**   +
 **Presentation**  +
 **Achievement, creativity** +
 **Ease of access** +

It was great how you you effectively used five of the five style elements, photo, italics, headers, list, and link. Moreover, you included the caption, which showed mastery of the elements needed for the homework, and from this, I learned have to do this. It was nice getting to read your biography, and knowing a bit about you. Great job!